### ISSUE TITLE
When installed in a docker container, the wasmtime path is not found

### ISSUE THREAD

**Issue by theagilecoder** [bytecodealliance/wasmtime#6524]
I am not facing this issue with wasmer or wasmedge, only wasmtime.
I hava simple dockerfile which installs wasmtime and prints its version 
```
# Use a base image with the desired Linux distribution
FROM ubuntu:latest

# Install necessary dependencies
RUN apt-get update && apt-get install -y curl

# Download and install Wasmer
RUN curl https://wasmtime.dev/install.sh -sSf | bash

# Add Wasmer binaries to the system PATH
ENV PATH="/root/.wasmtime/bin:${PATH}"

# Print the version of Wasmtime
RUN wasmtime --version

# Print Wasmer version
# CMD ["wasmtime", "--version"]
```

But it errors out saying wasmtime not found
`/bin/sh: 1: wasmtime: not found`

**Comment by bjorn3** [comment]
What is the value of `$HOME` when running the install command? It should install to `$HOME/.wasmtime/bin` if `$WASMTIME_HOME` is not set.

**Comment by theagilecoder** [comment]
I tried ENV PATH="$HOME/.wasmtime/bin:${PATH}" but get the same error. $HOME should be root. Please ELI5

**Comment by bjorn3** [comment]
What messages does the install script print?

**Comment by theagilecoder** [comment]
here are the portion of docker logs related to wasmtime install
```
[3/4] RUN curl https://wasmtime.dev/install.sh -sSf | bash
#6 1.363   Installing latest version of Wasmtime (v9.0.3)
#6 1.363     Checking for existing Wasmtime installation
#6 1.371     Fetching archive for Linux, version v9.0.3
#6 1.376 https://github.com/bytecodealliance/wasmtime/releases/download/v9.0.3/wasmtime-v9.0.3-x86_64-linux.tar.xz 
######################################################################## 100.0%        
#6 3.098     Creating directory layout
#6 3.103   Extracting Wasmtime binaries
#6 3.106 tar (child): xz: Cannot exec: No such file or directory
#6 3.106 tar (child): Error is not recoverable: exiting now
#6 3.107 tar: Child returned status 2
#6 3.107 tar: Error is not recoverable: exiting now
#6 3.110 cp: cannot stat '/tmp/tmp.oihoTMRyxk/wasmtime-v9.0.3-x86_64-linux/wasmtime': No such file or directory
#6 3.112 cp: cannot stat '/tmp/tmp.oihoTMRyxk/wasmtime-v9.0.3-x86_64-linux/LICENSE': No such file or directory
#6 3.112 cp: cannot stat '/tmp/tmp.oihoTMRyxk/wasmtime-v9.0.3-x86_64-linux/README.md': No such file or directory
#6 DONE 3.4s
```

**Comment by bjorn3** [comment]
Try installing xz.

**Comment by theagilecoder** [comment]
worked, thanks!
